pkg/ingester/ingester_test.go

```diff
@@ -6824,7 +6824,7 @@ func TestIngester_inflightPushRequests(t *testing.T) {
 		require.ErrorAs(t, err, &optional)
 		require.False(t, optional.ShouldLog(ctx, time.Duration(0)), "expected not to log via .ShouldLog()")
 
-		s, ok := status.FromError(err)
+		s, ok := grpcutil.ErrorToStatus(err)
```

Comment:
[nit]
Another possibility would be:

```suggestion
	code := grpcutil.ErrorToStatusCode(err)
```

and then to check 
```
require.Equal(t, codes.Unavailable, code)
```